package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-core-ui",
-    "version": "1.22.10",
+    "version": "1.22.11",
```

Comment:
**nitpick:** manual update of the version is not needed anymore. This will be processed automatically by the release tool based on the conventional commits. Please avoid continuing doing it manually, unless it is a backport, otherwise, it may conflict with the automated versioning.

Reply:
Done, thanks